packages/atomic/src/pages/examples/ipx.html

```diff
@@ -225,19 +279,24 @@
-          />
+          <a href="https://coveo.com/" target="_blank">
+            <atomic-icon
+              class="footer-icon"
```

Comment:
It might be better to use the SVG directly here, especially since `atomic-icon` tries to have a 1:1 aspect ratio by default and this icon isn't a square.